src/views/MainView/ImagesDropZone/ImagesDropZone.tsx

```diff
@@ -24,7 +23,9 @@ interface IProps {
 
 const ImagesDropZone: React.FC<IProps> = (props: PropsWithChildren<IProps>) => {
     const {acceptedFiles, getRootProps, getInputProps} = useDropzone({
-        accept: AcceptedFileType.IMAGE
+        accept: {
```

Comment:
Why not change the value under the enum? And use that enum value here?

Reply:
the value is object with params, I've tried to change it but haven't find good solution with the new dropzone api.

Reply:
I'll think about it